app/views/dashboard/_reports_widget.html.erb

```diff
@@ -14,13 +14,13 @@
     <tbody>
       <% events.each do |report| %>
         <tr>
-          <td><%= link_to trunc_with_tooltip(report.host, 14), host_config_reports_path(report.host) %></td>
-          <td><%= report_event_column(report.applied, "label-info") %></td>
-          <td><%= report_event_column(report.restarted, "label-info") %></td>
-          <td><%= report_event_column(report.failed, "label-danger") %></td>
-          <td><%= report_event_column(report.failed_restarts, "label-warning") %></td>
-          <td><%= report_event_column(report.skipped, "label-info") %></td>
-          <td><%= report_event_column(report.pending, "label-info") %></td>
+          <td class='ellipsis'><%= link_to report.host, host_config_reports_path(report.host) %></td>
```

Comment:
Chrome stable, here is how it looks like in my case:

![image](https://cloud.githubusercontent.com/assets/49752/12425573/acbf35ec-bed5-11e5-84a9-22f5c4fe1d5c.png)

Here is the content that does not fit:

```
Certificate: Data: Version: 3 (0x2) Serial Number: 2 (0x2) Signature Algorithm: sha256WithRSAEncryption Issuer: CN=Puppet CA: lzapx.brq.redhat.com Validity Not Before: Jul 10 13:21:45 2014 GMT Not After : Jul 10 13:21:45 2019 GMT Subject: CN=lzapx.brq.redhat.com Subject Public Key Info: Public Key Algorithm: rsaEncryption Public-Key: (4096 bit) Modulus: 00:d7:84:99:fe:fc:5b:82:f9:5b:0c:e8:83:a1:65: 5e:b8:55:60:28:ab:49:51:6f:36:f5:32:eb:79:7f: 03:74:bf:01:70:70:80:ea:93:71:83:85:19:3f:ef: bc:de:0f:e6:84:52:35:d3:53:ac:ce:c5:45:41:3d: 72:60:37:09:af:95:d1:0f:e7:db:1f:96:ef:a4:98: 16:d7:d7:90:5a:4d:76:21:00:b7:65:02:75:d2:e1: 1c:64:ec:2e:44:6f:22:64:c6:38:73:06:35:07:cc: 1d:39:14:3b:72:29:53:b2:14:77:39:62:42:e3:21: c1:78:62:b5:3f:80:02:57:32:53:cb:84:d0:99:7e: 4a:7c:07:79:f7:52:b4:35:59:40:a1:bf:7f:b0:14: d5:8b:ab:7b:91:3a:de:4a:8e:16:ed:8d:7b:db:2f: eb:a4:15:6a:84:1f:8c:e5:db:1d:a4:79:e3:c6:77: 1d:62:79:28:e3:43:54:1e:d9:29:3c:f5:28:62:17: 0d:a7:03:ff:97:4e:01:e9:74:36:32:dd:7a:c2:a3: 99:53:9c:a1:c7:e2:ef:db:42:65:a6:59:1d:f7:71: c8:07:7c:03:02:80:e8:ea:85:34:0b:f1:b8:2e:5c: 90:88:34:13:b8:36:de:83:c9:ef:b7:a9:e8:65:57: 2d:99:a9:b5:d2:3e:0d:b5:0d:e4:82:e8:3a:fd:45: c4:92:08:88:ca:b1:58:6a:8c:b3:87:76:e7:27:60: da:bc:c9:d0:12:40:67:b0:bf:8b:87:db:54:b0:c0: 31:ae:25:bb:cb:07:31:be:54:f2:93:96:15:78:b3: dc:7c:c9:e2:93:e6:a8:91:48:ca:d3:1b:da:10:b9: 5c:57:d2:64:2d:3a:29:44:bf:d7:66:79:3b:e1:f9: 8a:58:61:ba:8f:94:38:c8:e6:e8:28:96:a9:c9:1f: 30:f1:c9:af:59:2a:0a:60:1e:89:e1:74:6f:4e:52: 93:23:04:38:aa:81:8d:d7:e9:0f:02:f4:9d:04:d7: 54:39:48:3e:4a:7d:9a:5e:aa:92:1b:cb:94:36:31: eb:65:f3:a4:6c:de:a0:ed:19:9f:e7:8c:2f:a3:b1: 19:93:ad:ca:20:45:0e:e0:8f:8f:ec:7e:ff:d2:d0: 9b:f4:19:20:f1:59:e2:c7:95:e3:4b:45:51:b1:3c: eb:84:2f:60:66:ab:5a:ef:df:52:6a:1d:94:bd:54: 6b:ec:e9:69:7b:44:7b:3c:90:1a:af:bb:b0:7f:31: b8:fe:73:83:b1:97:ff:4f:8f:70:03:78:68:fc:d6: 73:46:60:f3:4b:a1:f8:61:45:d4:ae:f1:bc:3b:6f: 1e:fb:23 Exponent: 65537 (0x10001) X509v3 extensions: Netscape Comment: Puppet Ruby/OpenSSL Internal Certificate X509v3 Subject Alternative Name: DNS:lzapx.brq.redhat.com, DNS:puppet, DNS:puppet.brq.redhat.com X509v3 Key Usage: critical Digital Signature, Key Encipherment X509v3 Extended Key Usage: critical TLS Web Server Authentication, TLS Web Client Authentication X509v3 Basic Constraints: critical CA:FALSE X509v3 Subject Key Identifier: 58:FB:B9:7F:ED:AF:F8:6F:45:9D:78:BA:A6:F9:91:EE:7C:16:1A:35 X509v3 Authority Key Identifier: keyid:11:35:7C:0A:23:4D:3C:26:16:49:39:9F:4D:57:CE:58:C9:3C:4F:F8 Signature Algorithm: sha256WithRSAEncryption 59:3b:36:3a:7b:7e:a7:c8:fb:06:89:d7:25:5f:a9:dc:f6:68: 96:a8:f2:81:fb:4f:aa:38:9f:26:67:75:f3:4d:c6:db:44:4d: e8:58:9d:23:b0:73:c4:dc:d3:04:db:e4:20:f9:a2:65:14:38: 2d:fa:77:ef:63:4f:56:1e:1c:e7:61:db:dc:f5:9b:68:2f:3a: 9f:b7:a8:b5:66:b2:84:79:db:ac:ae:89:7c:d0:86:bd:41:cd: 2e:17:02:29:d2:73:95:75:29:fb:39:c4:f2:d5:82:2a:3e:6a: 19:e8:ee:e0:b3:4b:fe:af:03:44:28:bf:f5:01:b8:ac:22:ec: c4:8e:55:81:14:1d:e0:c4:45:c5:5a:e3:42:f0:fe:12:1d:2b: 77:0d:f7:35:26:91:c0:b5:3d:7b:b7:3a:dd:df:9a:24:1c:ef: e6:05:c4:4b:e9:cf:c6:73:b9:43:a1:22:5c:a5:8a:7e:fe:64: 35:f0:77:94:2f:73:76:27:08:db:35:d0:05:28:9d:47:8b:9d: dd:dc:7c:e3:b8:8d:5b:d8:04:1c:c1:f6:26:b5:76:f2:85:1b: 23:51:1c:40:0c:cf:2b:82:3f:a9:94:35:98:06:6a:35:5b:54: 76:3d:df:f8:f0:be:06:a7:61:7e:c4:15:bb:95:7d:75:bd:68: 18:c2:a2:c6:91:db:b8:25:ea:7f:56:76:3d:10:bd:ea:3a:66: c4:25:1f:93:f0:85:7c:ae:ff:44:6c:3c:a7:b6:30:d8:d5:28: 24:39:09:03:01:1c:85:73:c7:cb:b7:66:1c:5e:59:9a:f1:a8: 86:7c:d4:fb:12:58:a0:1e:89:c4:2f:c6:e9:70:81:9f:d5:cf: 2d:c3:87:42:bb:05:95:0e:e3:1c:3b:d2:cf:ca:99:d3:16:e5: ae:0d:e8:5d:48:f9:7c:d7:7a:bd:74:5d:4c:64:0a:0f:be:18: 33:aa:06:07:52:b0:22:7f:a7:11:1e:6b:11:05:b7:ff:c6:04: c8:91:7b:44:78:10:cd:76:31:20:84:9d:f8:23:1a:37:52:8a: 54:32:ec:a0:d4:15:a5:5e:38:9c:67:d5:41:bf:d1:72:16:6d: eb:ff:7e:f4:14:01:b8:1a:2c:95:6a:5e:68:6f:07:1c:eb:8d: ce:bc:09:bf:6a:de:67:a6:1a:73:ef:dd:64:3a:47:7d:e1:4b: ec:a9:ea:97:9d:b5:d4:38:a4:0b:1a:79:b1:74:a7:59:d1:43: 13:77:bc:e6:66:c6:e2:70:4a:c1:29:4d:fe:a4:69:4b:34:ae: 45:d2:d6:ce:01:f4:3a:51:86:81:70:03:f7:0b:ee:e6:6b:f7: 2e:cc:b0:0b:80:90:8f:d1
```